tests/auth_tests/urls.py

```diff
@@ -71,6 +71,8 @@ def custom_request_auth_login(request):
     url(r'^logout/next_page/named/$', views.logout, dict(next_page='password_reset')),
     url(r'^remote_user/$', remote_user_auth_view),
     url(r'^password_reset_from_email/$', views.password_reset, dict(from_email='staffmember@example.com')),
+    url(r'^password_reset_extra_email_context/$', views.password_reset,
+        dict(extra_email_context=dict(extra_email_context='Hello!'))),
```

Comment:
It would probably be less confusing if this were `dict(greeting='Hello!')` so we don't have so many parts called "extra_email_context".